### PR TITLE
Css+distancefilter

### PR DESCRIPTION
**Pull request by RoniNiklas**
- css altered to better match unity guidelines
- front page lottery can be filtered by distance
- filter is hidden by default, opened by button
-

### CODE REVIEW

File: packages/app/src/components/Filter/DistanceFilter/DistanceFilter.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+.distance-form-input {
```
Comment (ritinae) [inline packages/app/src/components/Filter/DistanceFilter/DistanceFilter.css]:
```suggestion
.distance-form-input {
```
Newline at top

File: packages/app/src/components/Filter/DistanceFilter/DistanceFilter.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    align-self:center;
+    margin: 0 !important;
+}
+.distance-form-input > input {
+    width: 7em !important;
+}
```
Comment (ritinae) [inline packages/app/src/components/Filter/DistanceFilter/DistanceFilter.css]: No newline at end
```suggestion
}

```

File: packages/app/src/components/Filter/DistanceFilter/DistanceFilter.test.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    )
+
+  const form = getByTestId(/distance-form/i)
+  const distanceField = within(form).getByTestId(/distance-field/i)
+  // Input test data
+  const distanceElement = within(distanceField).getByRole(/textbox/i)
+  fireEvent.change(distanceElement, { target: { value: newDistance } })
+  expect(mockSetDistance).toBeCalledWith(''+newDistance)
+})
+
```
Comment (ritinae) [inline packages/app/src/components/Filter/DistanceFilter/DistanceFilter.test.js:48]: Add tests for negative/invalid distance values

File: packages/app/src/components/Filter/FilterType/FilterType.css [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.filtertype-dropdown {
+    flex: 1;
+}
```
Comment (ritinae) [inline packages/app/src/components/Filter/FilterType/FilterType.css]:
```suggestion
}

```
No newline at end

File: packages/app/src/components/Randomizer/Randomizer.css [hunk 5]
```diff
@@ -10,10 +10,10 @@
...
-
-.restaurant-url:hover {
-    color: #E7A0E3;
+.randomizer-showFilterButton {
+    margin: 0.5em;
+    background-color: Transparent;
+    outline:none;
+    color:white; 
+    border:none;
 }
```
Comment (ritinae) [inline packages/app/src/components/Randomizer/Randomizer.css]:
```suggestion
}

```
No newline at end

**Review by ritinae — CHANGES_REQUESTED** [review]
Missing a few test cases

**Review by ritinae — APPROVED** [review]
